File: clients/WinForms/OpenWasherHardwareLibrary/Enums/PacketType.cs
namespace OpenWasherHardwareLibrary.Enums
{
    public enum PacketType
    {
        Ping = 0x00,
        CommandStart = 0x01,
        CommandStop = 0x02,
        GetDefaultOptions = 0x03,
        EnterProgrammingMode = 0x0A,
        GetStatus = 0x10,
        GetSetConfig = 0x20,
        GetSetTime = 0x30,
        Event = 0x5E,
        Error = 0x5B,
        Message = 0x58,
    };
}

File: clients/Flutter/openwasher/lib/Dto/PacketType.dart
class PacketType {
  static const Ping = 0x00;
  static const CommandStart = 0x01;
  static const CommandStop = 0x02;
  static const GetDefaultOptions = 0x03;
  static const EnterProgrammingMode = 0x0A;
  static const GetStatus = 0x10;
  static const GetSetConfig = 0x20;
  static const GetSetTime = 0x30;
  static const Event = 0x5E;
  static const Error = 0x5B;
  static const Message = 0x58;
}
